### PR TITLE
clang-format aes_icm_xxx.c

### PR DESCRIPTION
**Pull request by pabuhler**
part of issue #254

### CODE REVIEW

File: crypto/cipher/aes_icm.c [hunk 1]
```diff
@@ -198,21 +200,22 @@ static srtp_err_status_t srtp_aes_icm_context_init (void *cv, const uint8_t *key
...
-    debug_print(srtp_mod_aes_icm,
-                "offset: %s", v128_hex_string(&c->offset));
+    debug_print(srtp_mod_aes_icm, "key:  %s",
+                srtp_octet_string_hex_string(key, base_key_len));
+    debug_print(srtp_mod_aes_icm, "offset: %s", v128_hex_string(&c->offset));
 
     /* expand key */
-    status = srtp_aes_expand_encryption_key(key, base_key_len, &c->expanded_key);
+    status =
+        srtp_aes_expand_encryption_key(key, base_key_len, &c->expanded_key);
```
Comment (pabuhler) [inline crypto/cipher/aes_icm.c]: Not sure about this one, would have thought the assignment should stay on the same line but the function arguments break. @thisisG  what do you think?

Comment (thisisG) [reply crypto/cipher/aes_icm.c]: We can either play around with

PenaltyBreakAssignment (unsigned)
and
PenaltyBreakBeforeFirstCallParameter (unsigned)

or we can accept this as a quirk.

I am fine with either, let me know if we should have a stab at fixing it!

Comment (paulej) [reply crypto/cipher/aes_icm.c]: Personally, I like the way it turned out.

File: crypto/cipher/aes_icm.c [hunk 2]
```diff
@@ -291,10 +293,11 @@ static void srtp_aes_icm_advance (srtp_aes_icm_ctx_t *c)
  *  - fill buffer then add in remaining (< 16) bytes of keystream
  */
 
-static srtp_err_status_t srtp_aes_icm_encrypt (void *cv,
-                                               unsigned char *buf, unsigned int *enc_len)
+static srtp_err_status_t srtp_aes_icm_encrypt(void *cv,
```
Comment (paulej) [inline crypto/cipher/aes_icm.c]: Notices an "e" out of place a few lines above (line 283 in the new code).  That's not from these changes, but ... wanna fix it?

Comment (pabuhler) [reply crypto/cipher/aes_icm.c]: done

File: crypto/cipher/aes_icm.c (hunk 1 above)
Comment (pabuhler) [reply crypto/cipher/aes_icm.c]: It is not my personal preference but if it is not to common then it is ok. Will keep it like this for now and we can see what happens with the rest.

**Pull request merged**